variables.tf

```diff
@@ -32,6 +32,12 @@ variable "datadog_python_layer_version" {
   default     = 106
 }
 
+variable "datadog_enable_fips" {
```

Comment:
Could you rename it as `datadog_is_fips_enabled` to be consistent with Serverless Plugin?
https://github.com/DataDog/serverless-plugin-datadog/pull/598#discussion_r2109363052

Reply:
Done! I see that there was also a change in that PR to make FIPS the default when in GovCloud. Would you prefer that behavior here as well? I'd just need to move that logic into the `locals` block.

Reply:
Apologies for the late response! So in the serverless plugin `site` is a proper parameter, whereas here it's just a `DD_SITE` entry in an `environment_variables` map. That plus the fact that TF doesn't support conditional variable defaults (iirc) might make the implementation a bit sloppy.

I went ahead and gave it a go, but I'm also happy to revert it if it's a little too magical. Changing the default behavior might also be considered a breaking change.